airflow-core/src/airflow/ui/src/queries/useTrigger.ts

```diff
@@ -63,6 +64,33 @@ export const useTrigger = ({ dagId, onSuccessConfirm }: { dagId: string; onSucce
   };
 
   const onError = (_error: unknown) => {
+    // Check if error is a 403 (Forbidden) status
+    if (isHttpError(_error, 403)) {
+      toaster.create({
+        description: translate(
```

Comment:
Maybe an easier way to handle all cases instead of doing some manual mapping is to just display the `description: error.message`.

Similarly to how we do it in other places, for instance `usePatchTaskInstance`. This way the backend error is shown to the user.